liblumen_alloc/src/erts/term/map.rs

```diff
@@ -47,6 +47,13 @@ impl Map {
         self.value.contains_key(&key)
     }
 
+    pub fn keys(&self) -> Vec<Term> {
```

Comment:
Is it correct for this function to be returning `Vec<Term>` rather than a list directly? That's the impression I got from reading some of the other code, but I just want to verify that this is the interface you're intending to expose

Reply:
Yes because making a list would require passing in the `Process` or `HeapAlloc` to store the `Cons` cells inside, so this way separates getting the data from putting it in a format in the process in case runtime or alloc internal code doesn't really need a list in the process.